app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ActiveRecord::Base
+	has_many :games
```

Comment:
i was determining the white or black on the game model relationship.. i was thinking a user can have many games and the specific game would determine the white or black color chance lines 4 & 5 in the game model